backend/api/models.py

```diff
@@ -1,22 +1,94 @@
 from django.db import models
 from django.contrib.auth.models import User
+import uuid
+from django.db.models import DateTimeField, BooleanField
+
 
 # Create your models here.
 
-class DummyPost(models.Model):
-    text = models.CharField(max_length=255)
+
+# model for the category of a post
+class PostCategory(models.Model):
+    name = models.CharField(max_length=100, primary_key=True)
 
     def __str__(self):
-        return self.text
+        return self.name
 
-class AuthorProfile(models.Model):
-    author = models.ForeignKey(User, on_delete=models.CASCADE)
-    github = models.CharField(max_length=255)
 
+# model for a comment
+class Comment(models.Model):
+    author = models.ForeignKey(User, related_name="comments", on_delete=models.CASCADE)
```

Comment:
Doesn't each comment need to have a foreign key to a post

Reply:
fixed (redid) all the flipped relationships